code/game/gamemodes/cult/cult.dm

```diff
@@ -109,6 +108,7 @@
 					possible_targets += player.mind
 		if(possible_targets.len > 0)
 			sacrifice_target = pick(possible_targets)
+			GLOB.sac_target = sacrifice_target
```

Comment:
Calling it isn't the hard part, I dont know how to add a var to a specific game ticker. I'll figure it out I guess.